lib/directories.zsh

```diff
@@ -1,10 +1,11 @@
-[[ "${OMZ_DIRECTORY_ALIASES:-true}" == "false" ]] && return
-
 # Changing/making/removing directory
 setopt auto_pushd
 setopt pushd_ignore_dups
 setopt pushdminus
 
+zstyle -s ':omz:directories' aliases load_aliases || load_aliases=true
```

Comment:
Let's use the following line:
```zsh
zstyle -T ':omz:directories' aliases || return
```
It'll be configured in `zshrc` as
```zsh
zstyle ':omz:directories' aliases no
```

Reply:
Change done.  Should the doc changes go in the FAQ?